libraries/typings-generator/src/StringValuesTypingsGenerator.ts

```diff
@@ -78,31 +106,50 @@ function convertToTypingsGeneratorOptions<TFileContents>(
       return;
     }
 
+    const {
+      exportAsDefaultInterfaceName: exportAsDefaultInterfaceNameOverride,
+      exportAsDefaultDocumentationComment: exportAsDefaultDocumentationCommentOverride,
+      typings
+    } = stringValueTypings;
+
     const outputLines: string[] = [];
-    const interfaceName: string = options.exportAsDefaultInterfaceName
-      ? options.exportAsDefaultInterfaceName
-      : EXPORT_AS_DEFAULT_INTERFACE_NAME;
+    const interfaceName: string =
+      exportAsDefaultInterfaceNameOverride ||
+      exportAsDefaultInterfaceName ||
+      EXPORT_AS_DEFAULT_INTERFACE_NAME;
     let indent: string = '';
-    if (options.exportAsDefault) {
+    if (exportAsDefault) {
+      const documentationComment: string | undefined =
+        exportAsDefaultDocumentationCommentOverride || exportAsDefaultDocumentationComment;
+      if (documentationComment) {
+        const documentationCommentLines: string[] = documentationComment.split(/\r?\n/);
```

Comment:
I think the `Text` class in node-core-library has a line splitting function?

Reply:
It doesn't. Probably would be useful, though.